src/utils/searchViewHelper.js

```diff
@@ -50,6 +53,22 @@ const searchViewHelper = {
       searchViewHelper.searchByRevisionOrEmail(repository, search);
     }
   },
+
+  handleClickOutsideInput(event) {
+    try {
+      if (
+        event.target.id === 'search-revision-input' ||
+        event.target.classList.contains('search-revision') ||
+        event.target.parentElement.classList.contains('search-revision')
+      ) {
+        store.dispatch(updateSearchIsFocused(true));
+      } else {
+        store.dispatch(updateSearchIsFocused(false));
+      }
+    } catch (TypeError) {
```

Comment:
throws type error if element clicked has no parent element

Reply:
It would be better to test for this specific case instead of catching the error. If for some reason you have a `TypeError` anywhere else (for example in the reducers that run when dispatching the action) this would hide it.

`(event.target.parentElement && event.target.parentElement.classList.contains('search-revision'))`

Reply:
> It would be better to test for this specific case instead of catching the error. If for some reason you have a `TypeError` anywhere else (for example in the reducers that run when dispatching the action) this would hide it.

after using `matches` from your previous comment, it appears I no longer need to check for an error :ok_hand: